aana/projects/chat_with_video/endpoints.py

```diff
@@ -162,16 +176,16 @@ async def run(
             update_video_status(media_id=media_id, status=Status.RUNNING)
             audio: Audio = extract_audio(video=video_obj)
 
-            vad_output = await self.vad_handle.asr_preprocess_vad(
-                audio=audio, params=vad_params
-            )
-            vad_segments = vad_output["segments"]
+            # vad_output = await self.vad_handle.asr_preprocess_vad(
```

Comment:
Maybe add a comment sayin we are keeing this in for batch whisper changes?

Reply:
Done